source/contributing/doing-a-release.rst

```diff
@@ -83,19 +84,19 @@ Once the release branch has been thoroughly tested and is stable a release can b
 
     # Then commit the new version
     git add .
-    git commit -m "Update version to $newversion"
+    git commit -m "Update version to $newversionwithminor"
 
     # Push the release tag
-    git tag -a $version -m "Tag for $version release"
-    git push origin $version
+    git tag -a $newversion -m "Tag for $newversion release"
```

Comment:
Here we should also change to make tag, then build, then release to have same commit in build and tag (not related to the initial goal but good to update too if you've time)